.scss-lint.yml

```diff
@@ -90,10 +90,11 @@ linters:
     max_depth: 2
     severity: warning
   SelectorFormat:
-    enabled: true
-    convention: hyphenated_lowercase
+    enabled: false # strict_BEM doesn't seem to be supported by Hound
+    convention: strict_BEM
```

Comment:
Yeah it does, but it's not super accurate I'm seeing (e.g. doesn't seem to like the `.block--modifier` format. So I'm just going to keep it off.